docs/infrastructure/chromatic.md

```diff
@@ -0,0 +1,35 @@
+# Chromatic
+
+## What is Chromatic?
+
+Chromatic automates gathering UI feedback, visual testing, and documentation, so developers can iterate faster with less manual work.
+<https://www.chromatic.com>
+
+## What does Chromatic do?
+
+- Generates a unique public Storybook URL for each pull request
+- Detects visual changes in Storybook components
+- Allows manual inspection of Storybook component changes
+
+## Automatic Publish to Chromatic
+
+The repository has a github action for automatically publishing a branch to Chromatic on every Pull Request that changes the package `storybook` or a package dependency like `react-composites`
+
+This Github action is called `Publish Chromatic`.
+
+Checkout the [`.yml` file](https://github.com/Azure/communication-ui-library/blob/main/.github/workflows/publish-chromatic.yml) or the [github action](https://github.com/Azure/communication-ui-library/actions/workflows/publish-chromatic.yml) for more information.
```

Comment:
Can you use relative link here, so that the correct version of the file is linked on branches.